dashboard/app/controllers/projects_controller.rb

```diff
@@ -2,7 +2,7 @@
 require 'cdo/firehose'
 
 class ProjectsController < ApplicationController
-  before_action :authenticate_user!, except: [:load, :create_new, :show, :edit, :readonly, :redirect_legacy, :public, :index, :export_config, :weblab_footer]
+  before_action :authenticate_user!, except: [:load, :create_new, :show, :edit, :readonly, :redirect_legacy, :public, :index, :export_config, :weblab_footer, :get_or_create_for_level, :get_or_create_for_script_level]
```

Comment:
does `get_or_create_for_script_level` still need to be here?

Reply:
good catch!